nodejs/node
# test: add `--trace-gc` flag test

Hi 👋 

## Context

This pull request follows [this previous one](https://github.com/nodejs/node/pull/42346) and the [initiative](https://github.com/nodejs/diagnostics/issues/532) started by the diagnostic wg.

## Added

Now that `--trace-gc` is a part of the diagnostic tooling support, I thought that we could improve maturity tiers by adding a simple high-level test. As `--trace-gc` is a part of the v8 codebase, it probably doesn't make sense to test specific cases here. I just tried to test the integration between node and v8.

I first tried to find where this integration was made, but I didn't find it. While I read the code I expected to find it somewhere around here: https://github.com/nodejs/node/blob/de163d5db671fcad4ff6f98029c4de90ab4ff873/src/node_options.cc#L238 

If someone has an idea about where the integration is made, let me know ^^

**How to run this test?** -> `make test-v8-updates `

I wasn't sure about where should the test be or how I should write it, so I try to make it simple. 🙈
